tests/_utils.rs

```diff
@@ -79,7 +79,7 @@ impl Default for SampleWorkspace {
                 "0" => file!("")
                 "1" => file!("a")
                 "2" => file!("ab")
-                "3" => file!("abc")
+                "3" => file!("abcd")
```

Comment:
Let's make multiply these by an order of 1000. For example

```rust
      "3" => file!("a".repeat(3000))
```

This should be enough to avoid similar problems in other filesystems.

Reply:
Your suggested code actually resulted in a compilation error. I was able to make it work using the compiler's hints, but I'm not sure the result is ideal, since I don't know much Rust yet :D